README.md

```diff
@@ -107,13 +107,13 @@ Requirements: MonoGame + Visual Studio
 
 `MonoGame.Framework.OpenGL`
 
-9. Make sure you add Final Fantasy VIII path to array at `LinuxGameLocationProvider.cs:18`
+9. Make sure you add the Final Fantasy VIII path to the array at `LinuxGameLocationProvider.cs:18`
 
 10. That's all. You can now compile the executable.
 
 ## Development guidelines
 
-1. Project is in in-dev prototype, therefore you can make new pull requests directly to main branch. 
+1. Project is an in-dev prototype, therefore you can make new pull requests directly to main branch. 
```

Comment:
'in' in this case refers to a project status, rather than a product.

Reply:
True, and I think that "in" would be fine if it was written "Project is in dev", but here "in-dev" is an adjective modifying the noun "prototype", so we need an article.

Reply:
Maybe "This project is currently in active development, therefore..." would be better?